src/style.css

```diff
@@ -1,3 +1,71 @@
+html {
+  margin: 0;
+  padding: 0;
+
+  --hoverlink: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-size: 12px;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
+}
+
+.nav {
+  display: flex;
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  height: 60px;
+  list-style: none;
+}
+
+.nav__link {
+  display: inline-block;
+  margin-right: 20px;
```

Comment:
better:

.nav__link:not(:last-child) {
  margin-right: 20px;
}

Reply:
I know ,but it dont work . I made a new class for last link , sorry

Reply:
it's not the best solution, try to fix it with `:last-child` or ask this question on Q&A call